cypress/support/commands.js

```diff
@@ -329,8 +329,8 @@ Cypress.Commands.add('createUser', user => {
 	if (!('roles' in user)) user.roles = [];
 	cy.get('div[id=userGridContainer] a:contains("Add User")').click();
 	cy.wait(2000); // Avoid occasional glitches with given name field
-	cy.get('input[id^="givenName-en_US"]').type(user.givenName, {delay: 0});
-	cy.get('input[id^="familyName-en_US"]').type(user.familyName, {delay: 0});
+	cy.get('input[name="givenName[en_US]"').type(user.givenName, {delay: 0});
+	cy.get('input[name="familyName[en_US]"').type(user.familyName, {delay: 0});
```

Comment:
These lines are equivalent (the elements can be matched either way), so I don't think this changes anything...

Reply:
in the ids, I did not see the language attribute, therfore added it to the name.

 
```html
<input type="text" maxlength="255" required="true" validation="required" class="field text required" name="givenName[en_US]" value="" id="givenName-5e3d86cd81092" aria-required="true">
```

Reply:
Very strange -- that's different than I'm seeing. Do you have two languages installed and enabled (i.e. was the installation performed using the installation test, or did you perform it yourself)?

Reply:
I was  using the  cypress GUI and  I think, I may have missed some updates somehow, when merging from the pkp/pkp-lib.
Just did a fresh installation and a worked fine.